app/Http/Requests/EventRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class EventRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true; // Peut être ajusté selon vos besoins d'autorisation
+    }
+
+    public function rules()
+    {
+        return [
+            'title' => 'required|string',
+            'description' => 'required|string',
+            'location' => 'required|string',
+            'organized_by' => 'required|string',
```

Comment:
date debut date fin 
Titre 
Description
Secteur
Zone
Lieu dit
media
published_at

here are the possible attributes